Packs/MalwareInvestigationAndResponse/Scripts/KillProcessWrapper/KillProcessWrapper.py

```diff
@@ -0,0 +1,105 @@
+    """
+    Create a list of process kill commands (of `Cortex XDR`and `CrowdstrikeFalcon`), that should be executed
+    according to the given arguments.
+
+    Args:
+        endpoint_id: The Endpoint ID in which you would like to kill the given process.
+        process_id: The ID of the process to kill.
+        process_name: The name of the process to kill.
+
+    Returns: A list of Commands.
+    """
+    commands = []
+
+    if process_name and not process_id:
+        commands.append(CommandRunner.Command(commands=XDR_PROCESS_KILL_COMMAND,
+                                              args_lst={'endpoint_ids': endpoint_id, 'process_name': process_name}))
+    elif process_id and not process_name:
+        commands.append(CommandRunner.Command(commands=CROWDSTRIKE_PROCESS_KILL_COMMAND,
+                                              args_lst={'host_id': endpoint_id, 'process_ids': process_id}))
+    elif process_id and process_name:
+        # if a process ID and a process name were given, execute both the commands
+        commands.append(CommandRunner.Command(commands=XDR_PROCESS_KILL_COMMAND,
+                                              args_lst={'endpoint_ids': endpoint_id, 'process_name': process_name}))
+        commands.append(CommandRunner.Command(commands=CROWDSTRIKE_PROCESS_KILL_COMMAND,
+                                              args_lst={'host_id': endpoint_id, 'process_ids': process_id}))
+
+    return commands
+
+
+''' COMMAND FUNCTION '''
+
+
+def run_killing_process_action(endpoint_id: str = None,
+                               process_id: str = None,
+                               process_name: str = None) -> CommandResults:
+    """
+    Given arguments to the process kill command, returns a list including the results of the commands execution.
+
+    Args:
+        endpoint_id: The Endpoint ID in which you would like to kill the given process.
+        process_id: The ID of the process to kill.
+        process_name: The name of the process to kill.
+
+    Returns: A list of the kill process command results.
+    """
+    commands = create_commands(endpoint_id, process_id, process_name)
+    return CommandRunner.run_commands_with_summary(commands)
+
+
+''' MAIN FUNCTION '''
+
+
+def main():
+    args = demisto.args()
+    endpoint_id = args.get('endpoint_id')
+    process_id = args.get('process_id')
+    process_name = args.get('process_name')
+    approve_action = args.get('approve_action')
+
+    if approve_action == 'YES':
+        try:
+            # Handle missing arguments:
+            if not endpoint_id:
+                raise Exception('Endpoint ID was not specified.')
+            if not (process_id or process_name):
+                raise Exception('Process information was not specified - '
+                                'You should provide the process ID or the process name.')
+            return_results(run_killing_process_action(endpoint_id, process_id, process_name))
```

Comment:
Whats the case when we run both but only 1 installed, or 1 failed (as expected). This wont return as an error? I think we would only want to return an error if no successful commands ran

Reply:
We should also check, is there any message displayed after user clicks the button? (Action finished successfully, or action failed, popup)

Reply:
The readable output is displayed in the incident war room... I think that is satisfying.

Reply:
Thats how the other buttons currently work?